srml/contracts/src/rent.rs

```diff
@@ -165,6 +168,8 @@ fn try_evict_or_and_pay_rent<T: Trait>(
 			..contract
 		});
 
+		T::RentPayment::on_unbalanced(NegativeImbalanceOf::<T>::wrap(dues));
```

Comment:
```suggestion
		T::RentPayment::on_unbalanced(imbalance);
```

Reply:
and further up do `let imbalance = T::Currency::withdraw(..)`

Reply:
Oops! Thanks!